nens_auth_client/models.py

```diff
@@ -192,3 +211,26 @@ def send_email(self, request, context=None, send_email_options=None):
 
         self.email_sent_at = timezone.now()
         self.save()
+
+
+def clean_invitations(days=90):
+    """Delete invitations that are older than given amount of days.
+
+    Args:
+      days (int)
+
+    Returns:
+      the number of Invitations deleted (int)
+
+    Note that ``days`` must be >= NENS_AUTH_INVITATION_EXPIRY_DAYS, and
+    preferably significantly larger to be able to inform the user
+    that an Invitation has expired (as opposed to 'not found').
+    """
+    if days < settings.NENS_AUTH_INVITATION_EXPIRY_DAYS:
+        raise ValueError(
+            "The 'days' argument cannot be less than the Invitation expiry "
+            "setting ({}).".format(settings.NENS_AUTH_INVITATION_EXPIRY_DAYS)
+        )
```

Comment:
Kan dit beter in django's check mechanisme gestopt worden? Dan krijg je op het bruikbaarste moment feedback, niet als het het al gedeployed is.

Reply:
Probleem is dat t geen setting is, maar een argument van een management command

Reply:
Ah! Duidelijk.